src/main/CommandLine.cpp

```diff
@@ -939,20 +940,25 @@ runConsensusOnly(CommandLineArgs const& args)
 {
     CommandLine::ConfigOption configOption;
     uint32_t applicationSleepMs = 0;
+    uint32_t slotsToRemember = Herder::MAX_SLOTS_TO_REMEMBER;
 
-    return runWithHelp(args,
-                       {configurationParser(configOption),
-                        clara::Opt{applicationSleepMs, "MS"}["--spin-on-apply"](
-                            "simulate ledger application in ms")},
-                       [&] {
-                           Config cfg;
-                           cfg = configOption.getConfig();
-                           cfg.ARTIFICIALLY_SIMULATE_TX_APPLY_MS_FOR_TESTING =
-                               applicationSleepMs;
-                           cfg.ARTIFICIALLY_GENERATE_LOAD_FOR_TESTING = true;
-                           return runWithConfig(
-                               cfg, Application::AppMode::RUN_CONSENSUS_ONLY);
-                       });
+    return runWithHelp(
+        args,
+        {configurationParser(configOption),
+         clara::Opt{applicationSleepMs, "MS"}["--spin-on-apply"](
+             "simulate ledger application in ms"),
+         clara::Opt{slotsToRemember, "N"}["--slots-to-remember"](
```

Comment:
we don't need this, we can just expose `MAX_SLOTS_TO_REMEMBER` as a config setting